src/main/java/org/commonjava/indy/service/repository/change/audit/StoreAuditManager.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.repository.change.audit;
+
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.mapping.Mapper;
+import com.datastax.driver.mapping.MappingManager;
+import org.apache.commons.lang3.StringUtils;
+import org.commonjava.indy.service.repository.data.cassandra.CassandraClient;
+import org.commonjava.indy.service.repository.data.cassandra.CassandraConfiguration;
+import org.commonjava.indy.service.repository.data.cassandra.SchemaUtils;
+import org.commonjava.indy.service.repository.model.StoreKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+import static java.lang.String.format;
+
+@ApplicationScoped
+//@Startup
+public class StoreAuditManager
+{
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
+
+    public static final String TABLE_AUDIT = "audittable";
```

Comment:
Can we use another name without `table` ?  it's enough with just `audit` or `audit_log` or `repo_audit`.

Reply:
Sounds good. Fixed.